build.gradle

```diff
@@ -9,7 +9,7 @@ plugins {
 ext {
     sharedDir           = file("${project.rootDir}/shared")
 
-    sailfishVersion     = '3.2-SNAPSHOT'
+    sailfishVersion     = '3.2.1049'
```

Comment:
Don't we agree to build a new release for SF and use it?

Reply:
it's the latest release that we have (it was built less then week ago)